src/restc.erl

```diff
@@ -221,7 +221,9 @@ parse_response({ok, 204, Headers, Client}) ->
     ok = hackney:close(Client),
     {ok, 204, Headers, []};
 parse_response({ok, Status, Headers, Client}) ->
-    Type = parse_type(get_key(<<"Content-Type">>, Headers, ?DEFAULT_CTYPE)),
+    NormalizedHeaders = normalize_headers(Headers),
+    ContentType = content_type(NormalizedHeaders, ?DEFAULT_CTYPE),
+    Type = parse_type(ContentType),
```

Comment:
This crashes as well right? since content_type returns `{<<"Content-type">>, ContentType}` ...

Reply:
Yes hehe, I'm slowly getting there 😅